lib/lfrfid/tools/t5577.c

```diff
@@ -27,16 +33,13 @@ static void t5577_stop() {
 
 static void t5577_write_gap(uint32_t gap_time) {
     furi_hal_rfid_tim_read_pause();
-    furi_delay_us(gap_time * 8);
+    furi_delay_us(gap_time);
```

Comment:
Will be better to add something like `furi_delay_125k_clocks()` and count everything in field clocks.

Reply:
Added `furi_delay_clocks_125khz` in `furi/core/kernel.*`, idk if it's the right place

Reply:
No, not really, the 125kHz delay is only applicable to the LF-RFID subsystem and does not belong to the OS kernel.

Reply:
Ok, I renamed this function `t5577_furi_delay_clocks` and it was place in t5577.c as inline because it's basically just a 'rename' of furi_delay_us. I think we can move it out when we'll support in write more protocols other than t5577.